pkgs/development/python-modules/pikepdf/default.nix

```
@@ -81,6 +82,10 @@ buildPythonPackage rec {
     pillow
   ];
 
+  # lxml<5,>=4.8 not satisfied by version 5.1.0
+  # remove when python3.9 is removed
+  pythonRelaxDeps = lib.optional (pythonOlder "3.10") "lxml";
```

```suggestion
  pythonRelaxDeps = if (pythonOlder "3.10") then [ "lxml" ] else null;
```
We don't have to give up on avoiding mass rebuilds.

---

Isn't it what lib.optional does ? I will try anyways. Thank you for having a look

---

ok i've learned something today 
```
optional = cond: elem: if cond then [elem] else [];
```